src/marqo/tensor_search/api.py

```diff
@@ -41,12 +44,13 @@ def replace_host_localhosts(OPENSEARCH_IS_INTERNAL: str, OS_URL: str):
                 return replaced_str
     return OS_URL
 
-OPENSEARCH_URL = replace_host_localhosts(
-    os.environ.get("OPENSEARCH_IS_INTERNAL", None),
-    os.environ["OPENSEARCH_URL"])
-
+if __name__ in ["__main__", "api"]:
```

Comment:
This is needed to gate when api.py is imported, but not run as main script (e.g. unit tests).

Reply:
even during unit tests, it's still not run as the main script right?

Reply:
correct, but it imports functions from this file now (hence the need to change).

Reply:
Yeah, this is an explicit problem right now importing from `tensor_search/api.py`